src/components/tooltip/Marker.js

```diff
@@ -312,12 +312,15 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
         <TooltipDetailSeparator key="backtrace-separator" />,
         <TooltipDetail label="Stack" key="backtrace">
           <div className="tooltipDetailsBackTrace">
-            {data.type === 'Styles' || marker.name === 'Reflow' ? (
-              <h2 className="tooltipBackTraceTitle">
-                First invalidated {formatNumber(causeAge)}ms before the flush,
-                at:
-              </h2>
-            ) : null}
+            <h2 className="tooltipBackTraceTitle">
+              {data.type === 'Styles' || marker.name === 'Reflow'
+                ? `First invalidated ${formatNumber(
+                    causeAge
+                  )}ms before the flush,
+                at:`
+                : `Triggered ${formatNumber(causeAge)}ms ago,
```

Comment:
I'm more inclined to use `formatTimestamp` function here instead of `formatNumber`. That way we can remove the "ms" from here too  because that function prints the proper unit itself. I'm thinking about this because on some cases in the example profile, causeAge is pretty high like 2.5 seconds etc. So instead of printing a ms for a very high value, we can convert it to a second. Would you mind changing that?

I think we can keep the `formatNumber` for `Styles` and `Reflow` since it was implemented like this already. But it would be good to create an issue for changing that as well. Would you mind also creating an issue for converting this formatNumber above to the formatTimestamp?

Reply:
Sure 👍